### PR TITLE
Add /tracker command

### LINKED ISSUE #8: Move user text commands to slash commands (and maybe add context menu options)
!mute
!unmute
!setptt
!clearptt
!kick
!ban
!addguest (https://github.com/ClanAODDev/aod-discord-bot/issues/5)
!removeguest (https://github.com/ClanAODDev/aod-discord-bot/issues/5)
... others? Need to check

May make sense to isolate these under a single root `/user` or `/member` to keep the top level better organized

### PR DESCRIPTION
**Pull request by bashtech**
Adds the following slash commands:

- /tracker division [abbreviation]
- /tracker search [username]
- /tracker search-discord [user]
- /tracker search-teamspeak [unique-id]

Partially closes #8

### CODE REVIEW

File: commands/tracker.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+				method: 'post',
+				body: data,
+				headers: {
+					'User-Agent': 'Discord Bot',
+					'Accept': 'application/json'
+				}
+			});
+			let body = await response.json();
+			if (body.embed)
+				return global.messageReply(interaction, { embeds: [body.embed] });
```
Comment (apschultz) [inline commands/tracker.js:73]: Might be better to use ephemeralReply for tracker queries.

Comment (flashadvocate) [reply commands/tracker.js:73]: I will say, some NCOs will use the tracker search as a quick and easy way to point to someone they want something done for. IE "Someone go help go JoeSnuffy with this thing" "!tracker search:joesnuff"

Comment (flashadvocate) [reply commands/tracker.js:73]: So keeping the visible output might be helpful

Comment (bashtech) [reply commands/tracker.js:73]: I see some divisions use it to link to Tracker profiles in their officer channel in the current implementation so I didn't want to break this use case.

Comment (apschultz) [reply commands/tracker.js:73]: Works for me.

File: commands/tracker.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			else if (body.text)
```
Comment (apschultz) [inline commands/tracker.js:74]: Can we have an else with some default reply in case the tracker is borked?

Comment (bashtech) [reply commands/tracker.js:74]: Added in 4b07f66

**Review by apschultz — APPROVED** [review]

**Pull request merged**